VirtoCommerce.Platform.Web/Scripts/app/security/login/login.tpl.html

```diff
@@ -38,6 +38,11 @@
                     {{authError}}
                 </div>
             </div>
+            <div class="form-group">
```

Comment:
I think we shouldn't display this action if  `VirtoCommerce:Authentication:AzureAD.Enabled` setting is off